.github/workflows/community_ci.yml

```diff
@@ -42,12 +42,13 @@ jobs:
 
       # This will sync the .github folder of the main repo with the fork
       # This allows us to use up to date actions from the main repo
-      - name: Pull .github folder from main repository
+      - name: Pull .github folder and internal packages from main repository
         id: pull_github_folder
         run: |
           git remote add main https://github.com/airbytehq/airbyte.git
           git fetch main ${MAIN_BRANCH_NAME}
           git checkout main/${MAIN_BRANCH_NAME} -- .github
+          git checkout main/${MAIN_BRANCH_NAME} -- airbyte-ci
```

Comment:
This will make it impossible to test any changes to airbyte-ci itself, and I don't think this is necessary.